glide-core/redis-rs/redis/src/cluster_routing.rs

```diff
@@ -259,9 +374,19 @@ pub(crate) fn combine_and_sort_array_results<'a>(
         }),
         Value::Nil,
     );
-    assert_eq!(values.len(), sorting_order.len());
+    if values.len() != result_indices.len() {
+        return Err(RedisError::from((
+            ErrorKind::ClientError,
+            "Mismatch in the number of multi-slot results compared to the expected result count.",
+            format!(
+                "Expected: {:?}, Found: {:?}\nvalues: {values:?}\nresult_indices: {result_indices:?}",
```

Comment:
I'm not sure that it is safe to dump values into the log.

Reply:
these logs are client-side logs which are being saved on the user end